src/styles/app/03-app-shell.css

```
@@ -2,28 +2,25 @@
    APP SHELL - Main Layout with floating panels
    ───────────────────────────────────────────────────────────────────────────── */
 .appShell {
-	--app-frame-inline: var(--space-1);
-	--app-frame-block: 4px;
-	--app-radius: var(--radius-15);
+	--app-frame-inline: 0px;
+	--app-frame-block: 0px;
+	--app-radius: 0px;
 	--window-control-toggle-offset: 90px;
 	isolation: isolate;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)